query/src/interpreters/interpreter_factory_v2.rs

```diff
@@ -43,8 +44,8 @@ impl InterpreterFactoryV2 {
         matches!(stmt, DfStatement::SeeYouAgain)
     }
 
-    pub fn get(ctx: Arc<QueryContext>, plan: &Plan) -> Result<InterpreterPtr> {
-        let inner = InterpreterFactoryV2::create_interpreter(ctx.clone(), plan)?;
+    pub fn get(ctx: Arc<QueryContext>, plan: &Plan, raw_plan: &Plan) -> Result<InterpreterPtr> {
```

Comment:
Maybe we should wrap a new struct, because we need to support `explain sytnax/ast` in #7063  cc @b41sh 

```
struct PlanStmt {
 raw_plan: Plan,
 plan: Plan,
 ast:  Statement
}
```

Reply:
Make sense,  in this PR or in @b41sh  PR?

Reply:
Better to have a new struct in this pr, thus @b41sh  can work based on this.

Reply:
done